### PR TITLE
Modify documentation for stable storage

### PR DESCRIPTION
**Pull request by Kartik494**
Fixes: #28999

### CODE REVIEW

File: content/en/docs/concepts/workloads/controllers/statefulset.md [hunk 1]
```diff
@@ -173,8 +173,7 @@ Cluster Domain will be set to `cluster.local` unless
 
 ### Stable Storage
 
-Kubernetes creates one [PersistentVolume](/docs/concepts/storage/persistent-volumes/) for each
-VolumeClaimTemplate. In the nginx example above, each Pod will receive a single PersistentVolume
+Per each StatefulSet triggered pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object.In the nginx example above, each Pod will receive a single PersistentVolume
```
Comment (jihoon-seo) [inline content/en/docs/concepts/workloads/controllers/statefulset.md:176]: - `pod` -> `Pod`
- Added line breaks
  - (As-is: There is no space between `object.` and `In`)

```suggestion
Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object 
for each VolumeClaimTemplates entry defined in the StatefulSet object.
In the nginx example above, each Pod will receive a single PersistentVolume
```

**Review by jihoon-seo — COMMENTED** [review]
@Kartik494 PTAL!

File: content/en/docs/concepts/workloads/controllers/statefulset.md (hunk 1 above)
Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md:176]: @jihoon-seo Thanks for the review. I will be updating this

Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: @jihoon-seo Addressed the Changes. PTAL!

Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: Hi @jihoon-seo all the objectives for the required changes are addressed, Please let me know for any further changes if required.
Thanks

**Comment by jihoon-seo** [comment]
It will be great if someone performs an technical review on this PR.

**Comment by Kartik494** [comment]
@jihoon-seo Thanks for the response . Could you let me know whom to ping regarding the review.

File: content/en/docs/concepts/workloads/controllers/statefulset.md [hunk 2]
```diff
@@ -173,8 +173,7 @@ Cluster Domain will be set to `cluster.local` unless
 
 ### Stable Storage
 
-Kubernetes creates one [PersistentVolume](/docs/concepts/storage/persistent-volumes/) for each
-VolumeClaimTemplate. In the nginx example above, each Pod will receive a single PersistentVolume
+Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object. In the nginx example above, each Pod will receive a single PersistentVolume
```
Comment (celestehorgan) [inline content/en/docs/concepts/workloads/controllers/statefulset.md:176]: The grammar in the original issue wasn't fantastic. Try...

```suggestion
For each VolumeClaimTemplate entry defined in a StatefulSet, each Pod receives one PersistentVolumeClaim. In the nginx example above, each Pod receives a single PersistentVolume
```

Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: Hi i have addressed the changes PTAL!

Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: @celestehorgan Please let me know if any other changes are required in this PR.

Thanks!

Comment (Kartik494) [reply content/en/docs/concepts/workloads/controllers/statefulset.md]: Hi @jihoon-seo Could you Please take a look here. I have addressed the changes as suggested

**Pull request merged**